app/api/search.v2/buildQuery.ts

```diff
@@ -3,11 +3,12 @@ import { RequestBody } from '@elastic/elasticsearch/lib/Transport';
 import { cleanUp, extractSearchParams, snippetsHighlight } from './queryHelpers';
 import { permissionsFilters } from './permissionsFilters';
 
+const defaultFields = ['title', 'template', 'sharedId'];
 export const buildQuery = async (query: SearchQuery, language: string): Promise<RequestBody> => {
   const { searchString, fullTextSearchString, searchMethod } = await extractSearchParams(query);
   return {
     _source: {
-      includes: ['title', 'template', 'sharedId', 'language', 'documents._id'],
+      includes: query.fields || defaultFields,
```

Comment:
Isn't it needed to remove pseudofields like `snippets`?

Reply:
We could, but it's not needed. Elastic ignore all the fields that don't exist in the index. That's one of the reasons why at this moment we don't need to validate or sanitize this information because it would imply fetching the valid properties from the DB. Eventually in the future, if we want to apply highlights in some fields we will need to pre-process these values.